src/modules/location/templates/location.3d.handlebars

```diff
@@ -4,23 +4,23 @@
         <meta charset="utf-8">
         <meta http-equiv="'X-UA-Compatible' content='IE=edge'">
         <script src="https://aframe.io/releases/0.9.2/aframe.min.js"></script>
-        <script src="https://raw.githack.com/jeromeetienne/AR.js/master/aframe/build/aframe-ar.min.js"></script>
-        <script src="https://raw.githack.com/donmccurdy/aframe-extras/master/dist/aframe-extras.loaders.min.js"></script>
+        <script src="https://unpkg.com/aframe-look-at-component@0.8.0/dist/aframe-look-at-component.min.js"></script>
+        <script src="https://raw.githack.com/AR-js-org/AR.js/master/aframe/build/aframe-ar-nft.js"></script>
         <script>
             THREEx.ArToolkitContext.baseURL = 'https://raw.githack.com/jeromeetienne/ar.js/master/three.js/'
         </script>
     </head>
 
     <body style="margin: 0; overflow: hidden;">
         <a-scene
+            renderer="logarithmicDepthBuffer: true;"
             vr-mode-ui="enabled: false"
-            arjs="sourceType: webcam; sourceWidth:1280; sourceHeight:960; displayWidth: 1280; displayHeight: 960; debugUIEnabled: false;"
+            arjs="sourceType: webcam; debugUIEnabled: false;"
             embedded
         >
             <a-entity
                 gltf-model="{{assetSrc}}"
-                rotation="0 180 0"
-                scale="1 1 1"
+                look-at="[gps-camera]"
```

Comment:
Is this a parameter that we should handle with handlebar? Or it's a normal string?

Reply:
I added the import of that component, 'look-at', and the default use of it. It is intended to be use as default for location based, in my opinion. We should keep it, at least for now :)

Reply:
So to answer you, I would use it as I added it, hardcoded.